modules/data.land/R/extract_NEON_veg.R

```diff
@@ -163,7 +164,7 @@ extract_NEON_veg <- function(lon, lat, start_date, end_date, store_dir, neonsite
         joined.soil$bulkDensity <- bulkDensity
         
         #convert from g/cm2 to g/m2, note that we have to divide by 100 because of percentage
-        joined.soil$SoilCarbon <- (joined.soil$organicCPercent * joined.soil$bulkDensity)*30*100 
+        joined.soil$SoilCarbon <- PEcAn.utils::ud_convert((joined.soil$organicCPercent * joined.soil$bulkDensity)*30*100, 'g m-2', 'kg m-2')#convert from g to kg.
```

Comment:
Comment says we have to divide by 100 but then the code multiplies by 100, which is confusing. Also unclear what the 30 is here.